plugins/events-backend-module-sqs/src/service/AwsSqsEventsModule.ts

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2022 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+  configServiceRef,
+  createBackendModule,
+  loggerServiceRef,
+  loggerToWinstonLogger,
+  schedulerServiceRef,
+} from '@backstage/backend-plugin-api';
+import { eventsExtensionPoint } from '@backstage/plugin-events-node';
+import { AwsSqsConsumingEventPublisher } from './AwsSqsConsumingEventPublisher';
+
+/**
+ * AWS SQS module for the Events plugin.
+ *
+ * @alpha
+ */
+export const awsSqsEventsModule = createBackendModule({
```

Comment:
should it be clear that it's the consuming one? as opposed to the one that pushes stuff out of the bus to sqs :P Then again, that could be served by one and the same module, just configured differently (or both ways)

Reply:
I renamed it following the names of other modules